frontend/src/utils/layers-utils.tsx

```diff
@@ -375,7 +398,9 @@ const useLayers = () => {
   const possibleDatesForLayerIncludeSelectedDate = useCallback(
     (layer: DateCompatibleLayer, date: Date) => {
       return binaryIncludes<DateItem>(
-        getPossibleDatesForLayer(layer, serverAvailableDates),
+        layer.type === 'anticipatory_action'
```

Comment:
Let's add a comment here explaining why anticipatory_action is handled differenlty